### ISSUE TITLE
Additional sources to expand cfkg with

### ISSUE THREAD

**Issue by dgarijo** [TEC-Toolkit/cfkg#10]
The NAEI (also from the UK) looks like an interesting source to expand the KG, possibly refining ECFO: https://naei.beis.gov.uk/data/ef-all-results?q=180219

**Comment by stefanogermano** [comment]
Yes, it was already in the list of resources reported in Discussion 5 😉

**Comment by dgarijo** [comment]
Expanding the ones in Discussion 5 with https://www.ipcc-nggip.iges.or.jp/EFDB/find_ef.php?ipcc_code=1.A.1.a.i&ipcc_level=4

**Comment by stefanogermano** [comment]
> Expanding the ones in Discussion 5 with https://www.ipcc-nggip.iges.or.jp/EFDB/find_ef.php?ipcc_code=1.A.1.a.i&ipcc_level=4

It seems to be the same as

> https://www.ipcc-nggip.iges.or.jp/EFDB/main.php

which was already mentioned in Discussion 5.

However, it is good that this resource has been found twice independently (it seems to be very interesting/important).
